web/app/components/inputs/people-select.ts

```diff
@@ -17,6 +17,8 @@ interface InputsPeopleSelectComponentSignature {
   Args: {
     selected: HermesUser[];
     onChange: (people: HermesUser[]) => void;
+    renderInPlace?: boolean;
+    disabled?: boolean;
```

Comment:
Happened to come across this missing types. SORRY

Reply:
![](https://media.giphy.com/media/v1.Y2lkPTc5MGI3NjExZHdjcW1oc3c0Zmp5OWs3Z3NndjRjenNoeWVnZjBmeXg3c2FrZm1ybSZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/eveLVPcHcbl0A/giphy-downsized.gif)